### PR TITLE
Update mandatory-user-profile.md

### PR DESCRIPTION
**Pull request by warren-msft**

### CODE REVIEW

File: windows/client-management/mandatory-user-profile.md [hunk 1]
```diff
@@ -94,6 +94,8 @@ First, you create a default user profile with the customizations that you want,
 5. The sysprep process reboots the PC and starts at the first-run experience screen. Complete the set up, and then sign in to the computer using an account that has local administrator privileges.
 
 6. Right-click Start, go to **Control Panel** (view by large or small icons) > **System** > **Advanced system settings**, and click **Settings** in the **User Profiles** section.
```
Comment (JohanFreelancer9) [inline windows/client-management/mandatory-user-profile.md:96]:
```suggestion
6. Right-click **Start**, go to **Control Panel** (view by large or small icons) > **System** > **Advanced system settings**, and click **Settings** in the **User Profiles** section. Alternatively, starting in Windows 10, version 2004, open the **Settings** app and select **Advanced system settings**.
```

File: windows/client-management/mandatory-user-profile.md [hunk 2]
```diff
@@ -94,6 +94,8 @@ First, you create a default user profile with the customizations that you want,
...
+
+Starting in Windows 10 version (2004) Open the Settings app and click on Advanced system settings
```
Comment (JohanFreelancer9) [inline windows/client-management/mandatory-user-profile.md:98]:
```suggestion
```

**Comment by illfated** [comment]
This PR targets the file `windows/client-management/mandatory-user-profile.md` which is the same file used in PR #9141 — likely reason for the file conflict.

File: windows/client-management/mandatory-user-profile.md [hunk 3]
```diff
@@ -103,12 +105,13 @@ First, you create a default user profile with the customizations that you want,
...
    
-9. In **Select User or Group**, in the **Enter the object name to select** field, type `everyone`, click **Check Names**, and then click **OK**.
+9. In **Select User or Group**, in the **Enter the object name to select** field, type `everyone` or the group of users that the profile will be assigned to, click **Check Names**, and then click **OK**.
 
 10. In **Copy To**, in the **Copy profile to** field, enter the path and folder name where you want to store the mandatory profile. The folder name must use the correct [extension](#extension) for the operating system version. For example, the folder name must end with “.v6” to identify it as a user profile folder for Windows 10, version 1607.
 
    - If the device is joined to the domain and you are signed in with an account that has permissions to write to a shared folder on the network, you can enter the shared folder path.
    - If the device is not joined to the domain, you can save the profile locally and then copy it to the shared folder location.  
+   - Optionally you can check the box “Mandatory profile”. This step is not required but will set permissions that are more restrictive and it is recommend to do so
```
Comment (JohanFreelancer9) [inline windows/client-management/mandatory-user-profile.md:114]:
```suggestion
   - Optionally, you can check the **Mandatory profile** checkbox. This step is not required but will set permissions that are more restrictive and we recommend doing so.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: windows/client-management/mandatory-user-profile.md (hunk 3 above)
Comment (illfated) [reply windows/client-management/mandatory-user-profile.md:114]: Maybe add a comma after "required", to have the sentence correctly divided into main sentence + clause, like this:
```suggestion
   - Optionally, you can check the **Mandatory profile** checkbox. This step is not required, but will set permissions that are more restrictive and we recommend doing so.
```

**Comment by e0i** [comment]
@warren-msft Please follow-up with the copy editing suggestions and if you can, resolve the merge conflicts to proceed. Thank you again for the contributions.

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/client-management/mandatory-user-profile.md [hunk 4]
```diff
@@ -53,6 +53,8 @@ First, you create a default user profile with the customizations that you want,
    > [!NOTE]
    > Use a lab or extra computer running a clean installation of Windows 10 to create a default user profile. Do not use a computer that is required for business (that is, a production computer). This process removes all domain accounts from the computer, including user profile folders. 
 
+=======
+
```
Comment (illfated) [inline windows/client-management/mandatory-user-profile.md:57]: This looks like a remnant from resolving the file conflict. I am fairly certain we don't need those 2 lines.
```suggestion
```

File: windows/client-management/mandatory-user-profile.md [hunk 5]
```diff
@@ -99,9 +102,14 @@ Starting in Windows 10 version (2004) Open the Settings app and click on Advance
 1. In **Copy To**, in the **Copy profile to** field, enter the path and folder name where you want to store the mandatory profile. The folder name must use the correct [extension](#profile-extension-for-each-windows-version) for the operating system version. For example, the folder name must end with ".v6" to identify it as a user profile folder for Windows 10, version 1607.
 
    - If the device is joined to the domain and you are signed in with an account that has permissions to write to a shared folder on the network, you can enter the shared folder path.
+
+   
+   ![Example of UI](images/copy-to-path.png) 
+=======
```
Comment (illfated) [inline windows/client-management/mandatory-user-profile.md:108]: Same here, a remnant from resolving the file conflict.
```suggestion

```

File: windows/client-management/mandatory-user-profile.md [hunk 6]
```diff
@@ -99,9 +100,13 @@ Starting in Windows 10 version (2004) Open the Settings app and click on Advance
 1. In **Copy To**, in the **Copy profile to** field, enter the path and folder name where you want to store the mandatory profile. The folder name must use the correct [extension](#profile-extension-for-each-windows-version) for the operating system version. For example, the folder name must end with ".v6" to identify it as a user profile folder for Windows 10, version 1607.
 
    - If the device is joined to the domain and you are signed in with an account that has permissions to write to a shared folder on the network, you can enter the shared folder path.
+
+   
+   ![Example of UI](images/copy-to-path.png) 
    - If the device is not joined to the domain, you can save the profile locally and then copy it to the shared folder location.
```
Comment (illfated) [inline windows/client-management/mandatory-user-profile.md:106]: Thank you for merging my suggestions. Just an image spacing issue remains, from what I can see:
```suggestion

   ![Example of UI](images/copy-to-path.png)

   - If the device is not joined to the domain, you can save the profile locally and then copy it to the shared folder location.
```

**Review by illfated — APPROVED** [review]
Looks good to me.

**Comment by illfated** [comment]
#sign-off

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by warren-msft** [comment]
I just checked the Docs content and the changes never got published.

**Comment by illfated** [comment]
Yes, unfortunately, it sometimes happens that the migration subroutine fails silently. Please advise on how to proceed to have this PR commit change migrated to docs.microsoft.com .

**Pull request merged**